File: hw/aimacana/u2/hw16/javascript/taxes-lib/tax.js
export class Tax {
    static computeIva(amount, taxPercentage) {
        return amount * (taxPercentage / 100.0);
    }

    static computeTotal(amount, taxPercentage) {
        return amount + Tax.computeIva(amount, taxPercentage);
    }
}

File: ws/nacato/u2/ws16Libraries/TaxesLib/src/utils/Tax.java
package utils;

/**
 *
 * @author Angie Nacato, Error 404, @ESPE
 */
public class Tax {
    public static float computeIva(float amount,float taxPercentage){
        float taxValue;
        taxValue = amount * taxPercentage/100;
        return taxValue;
    }
    public static float computeTotal(float amount, float taxPercentage){
        float totalValue;
        totalValue = amount + computeIva(amount, taxPercentage);
        return totalValue;
    }
}
